### PR TITLE
Add Manual Test Cases for `deleteNok` and `viewSchedule`

### PR DESCRIPTION
**Pull request by CraveToCode**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1029,14 +1029,28 @@ testers are expected to do more *exploratory* testing.
...
+### Delete all NoK details of an elderly.
+
+1. Deleting all NoK details of an elderly.
+
+    1. Prerequisites: List all elderlies using the `viewElderly` command. Multiple elderlies in the list.
+
+    2. Test case: `deleteNok 1` <br>
+       Expected: All NoK details of the first elderly in the list are deleted. Updated details of the elderly whose NoK details were deleted are shown in the status message.
+
+    4. Invalid commands to try (Error details shown in the status message):
```
Comment (tsy24) [inline docs/DeveloperGuide.md:1042]: Numbering should be 3?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1029,14 +1029,28 @@ testers are expected to do more *exploratory* testing.
...
+        * Invalid index >= size of elderly list, or <= 0: `deleteElderly 5` or `deleteElderly -1`
```
Comment (tsy24) [inline docs/DeveloperGuide.md:1043]: Commands here should be `deleteNok`?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -1101,8 +1115,34 @@ testers are expected to do more *exploratory* testing.
...
+   
+    4. Test case - four days ahead:`viewSchedule 2021-11-16` <br>
+       Expected: Tasks B and C should be displayed. Date of Task B in this display should be `2021-11-16`.
+   
+    5. Test case - a week ahead:`viewSchedule 2021-11-19` <br>
+       Expected: Task B should be displayed. Date of Task B in this display should be `2021-11-19`.
+   
+    6. Some other incorrect `viewSchedule` commands to try (error details shown in status message): 
+        * Date that has already passed: `viewSchedule 2021-11-11` <br> Expected: Program should output error that the date has already passed.
+        * Not formatting the date correctly: `viewSchedule 16-12-2021` <br> Expected: Program should output error that date should be in yyyy-mm-dd format.
```
Comment (tsy24) [inline docs/DeveloperGuide.md:1143]: Perhaps it would be better to follow the formatting above?

**Review by tsy24 — CHANGES_REQUESTED** [review]
Just some minor mistakes to change!

**Comment by CraveToCode** [comment]
Effort section will be added in a future PR!

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (CraveToCode) [reply docs/DeveloperGuide.md:1043]: Good catch! Will update

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (CraveToCode) [reply docs/DeveloperGuide.md:1042]: You're right!

**Review by tsy24 — APPROVED** [review]
LGTM!

**Pull request merged**